src/view/src/Include_i.h

```diff
@@ -60,6 +65,8 @@ typedef struct {
     // The actual buffer which follows immediately after the structure
     PViewItem itemBuffer;
 
+    CONTENT_VIEW_OVERFLOW_STRATEGY bufferOverflowStrategy;

```

Comment:
Super careful here! The entire structure and the array is crafted to be 64bit aligned. Move the bufferOverflowStrategy before itemBufferCount. Add comments. Make the variable UINT64 to not break the alignment. Cast-assign to UINT64. Another strategy is to use CONTENT_VIEW_OVERFLOW_STRATEGY type and pad with UINT32 reserved;